docs/built-in-models/encrypted_dataframe.md

```diff
@@ -0,0 +1,111 @@
+# Working with Encrypted DataFrames
+
+Concrete ML builds upon the pandas data-frame functionality by introducing the capability to construct and perform operations on encrypted data-frames using FHE. This API ensures data scientists can leverage well-known pandas-like operations while maintaining privacy throughout the whole process.
+
+Potential applications include:
+
+- Encrypted storage of tabular datasets
+- Joint data analysis efforts between multiple parties
+- Data preparation steps before machine learning tasks, such as inference or training
+- Secure outsourcing of data analysis to untrusted third parties
+
+## Encrypt and Decrypt a DataFrame
+
+To encrypt a pandas `DataFrame`, use the `encrypt_from_pandas` function. First, a client engine, which manages keys, must be constructed using `load_client`.
+
+```python
+from concrete.ml.pandas import ClientEngine, load_encrypted_dataframe
+from io import StringIO
+import pandas
+
+data_left = """index,total_bill,tip,sex,smoker
+1,12.54,2.5,Male,No
+2,11.17,1.5,Female,No
+3,20.29,2.75,Female,No
+"""
+
+# Load your pandas DataFrame
+df = pandas.read_csv(StringIO(data_left))
+
+# Obtain client object
+client = ClientEngine(keys_path="my_keys")
```

Comment:
no need to specify the key path in this example imo

Reply:
I think it's good to show that ClientEngine works with the keys